src/test/java/hudson/tasks/AntTest.java

```diff
@@ -356,9 +361,12 @@ private void testVariableReplaced(String variableValue) throws Exception {
         ParameterDefinition paramDef = new StringParameterDefinition("variable", "");
         ParametersDefinitionProperty paramsDef = new ParametersDefinitionProperty(paramDef);
         project.addProperty(paramsDef);
-
-        FreeStyleBuild build = project.scheduleBuild2(0, new Cause.UserIdCause(),
-                new ParametersAction(new StringParameterValue("variable", variableValue))).get();
+        
+        Set<String> safeParams = new HashSet<String>();
+        safeParams.add("variable");
```

Comment:
I do not get it. We already added a parameter definition for this, in #27. So what is the problem?